City-of-Helsinki/drupal-helfi-etusivu
# UHF-8516 news lander sitemap

# [UHF-8516](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8516)

## What was done
- Added etusivu sitemap to sitemap index
- Added news landing pages to sitemap (/uutiset, /news, /nyheter)


## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-8516_news_lander_sitemap`
  * `make fresh`
* Run `make drush-cr`

## How to test
- Run `drush cr`
- Run `drush ssg`
- Go to https://helfi-etusivu.docker.so/sitemap.xml (sitemap index)
  - Link to etusivu production sitemap should be set.
- Go to /fi/sitemap.xml
  - search for the news landing pages ('/fi//uutiset ', '/en/news ', '/sv/nyheter ')





[UHF-8516]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8516?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
